week-01/assessment-1/src/main/java/HotelApp.java

```diff
@@ -0,0 +1,343 @@
+import java.util.Random;
+import java.util.Scanner;
+public class HotelApp {
+
+    public static void main(String[] args) {
+    Scanner scn = new Scanner(System.in);
+        System.out.println("Welcome to Hotel 69.");
+        System.out.println("=====================");
+        int capsules = 0;
+//        int roomAvail = 0;
+
+        capsules = capsuleCount();
+
+        String[] capsulesArray = new String[capsules];
+        System.out.println("There are " + capsules + " unoccupied capsules ready to be booked.");
+
+
+
+     // int choice = Integer.parseInt(mainMenu());
+
+
+
+      int num = Integer.MIN_VALUE;
+
+      boolean isFinished = false;
+
+//            switch (choice) {
```

Comment:
When you're marking a project off as complete, take a moment to delete any unnecessary comments